src/navx_node.cpp

```diff
@@ -0,0 +1,106 @@
+/*
+ * navx_node.cpp
+ * Runs the Kauai Labs NavX, using modified NavX library
+ * VERSION: 0.0
+ * Last changed: 2019-04-01
+ * Authors: Jude Sauve <sauve031@umn.edu>
+ * Maintainers: Jude Sauve <sauve031@umn.edu>
+ * MIT License
+ * Copyright (c) 2018 GOFIRST-Robotics
+ */
+
+// ROS Libs
+#include <ros/ros.h>
+#include <sensor_msgs/Imu.h>
+
+// Native_Libs
+#include <string>
+
+// Custom_Libs
+#include "ahrs/AHRS.h"
+
+// Subscribers (inputs)
+//    update_timer (Timer)
+//      Update loop for reading / querying IMU
+
+// Publishers (outputs)
+//    imu_pub (sensor_msgs/Imu): imu/data
+//      The published imu data
+
+// Parameters (settings)
+//    frequency (double): default=50.0
+//      The update frequency of the update loop
+//    param_name2 (param_name2_type): default=param_name2_default(,param_name1_path)
+//    param_name3 (param_name3_type): default=param_name3_default(,param_name1_path)
+
+
+// ROS Node and Publishers
+ros::NodeHandle * nh;
+ros::Publisher imu_pub;
+
+// ROS Topics
+std::string imu_topic = "imu/data";
+
```

Comment:
Delete. It's slightly better form to just put the topics directly in the pub/sub in main.